L/libassuan/build_tarballs.jl

```diff
@@ -0,0 +1,47 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "libassuan"
```

Comment:
Also here, the package styles the name as
```suggestion
name = "Libassuan"
```